src/Stryker.Core/Stryker.Core/Mutants/MutationContext.cs

```diff
@@ -47,10 +56,28 @@ public MutationContext Clone()
             return new MutationContext(this);
         }
 
+        public void Discard()
+        {
+            if (HasStatementLevelMutant)
+            {
+                // some mutants 
+                Logger.LogInformation($"{BlockLevelControlledMutations.Count+StatementLevelControlledMutations.Count} mutations were not injected.");
```

Comment:
It seems like this might be logged often. Is this important for the user to know about? If so there should be some clarification on what the user should do with this information. If not this should at a minimum be LogDebug.

Reply:
This should not happen often. Currently, AFAIK it only happens with array initializers in fields, which result in mutations that cannot be injected.
I will put it at debug level for the time being.

Reply:
see PR #1530.
https://github.com/stryker-mutator/stryker-net/pull/1530/files#diff-41b1bf6c0a66c50dad49e7ad055c28211a76f2e663cd9bcb9fcacc1477e98b0fR64